### DISCUSSION TITLE
Is server down?

### DISCUSSION THREAD

**Discussion by Hosung99** [apexcharts/apexcharts.js#4820] [Q&A]
hi i'm korean user.
Maybe your site error occurred from 10 o'clock Korean time. Please check if there is a problem.
<img width="821" alt="image" src="https://github.com/user-attachments/assets/f743247f-21a1-4870-9a34-e53193fbdd5c">

**Comment by slipperystairs** [comment]
It is most certainly down.

**Comment by junedchhipa** [comment]
It is back now

**Comment by Hosung99** [comment]
thanks for notice.
